src/main/java/objects/battle/StatDisplayer.java

```diff
@@ -0,0 +1,38 @@
+package objects.battle;
+
+import io.Output;
+import io.OutputHandler;
+import objects.character.EnemyFacade;
+import objects.character.Player;
+
+public class StatDisplayer {
+    /**
+     * Attributes:
+     * output: Screen that allows for output
+     * user: Player object whose stats are being referenced
+     * foe: Enemy object whose stats are being referenced
+     */
+    private final OutputHandler output = Output.getScreen();
+    public StatDisplayer() {
+
+    }
+
+    /**
+     * Displays the stats of the user and the enemy.
+     */
+    public void displayStats(Player user) {
```

Comment:
Having a class for this makes the code very clean in terms of the clean architecture so I think it is very good idea